react-carousel/src/tools/simulateEvent.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
what's the point if the `react-carousel` directory?

Reply:
Do you mean react-carousel workspace? I use it because docs-www and react-carousel shared the same dependencies which were installed twice in carousel v1. With workspaces, common dependencies are installed once

Reply:
ok